buildenv/jenkins/testJobTemplate

```diff
@@ -361,8 +361,8 @@ ARCH_OS_LIST.each { ARCH_OS ->
                                                                        individual test in playlist.xml: floatSanityTests|MiniMix_5m|jdk_math|jdk_beans_0|jdk_custom<br/>
                                                                        a list of tests: testList TESTLIST=jit_jitt,jit_recognizedMethod_1,testSCCMLTests2_1<br/>
                                                                        Note: please do not use level, group or type (i.e., functional, sanity, etc) in the TESTLIST<br/>''')
-							stringParam('CUSTOM_TARGET', "", '''
-								Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, system_custom, etc., CUSTOM_TARGET=path to the test class to execute<br>
+							stringParam('CUSTOM_TARGET', "",
+								'''Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, system_custom, etc., CUSTOM_TARGET=path to the test class to execute (typically ending in .java or .sh)<br>
```

Comment:
This is statement is not incorrect, but doesn't cover several other cases, so may mislead people into thinking CUSTOM_TARGET can only be used for a single test class (when it can also be used to point to a directory or one can also pass a list of test classes separated by spaces).

Reply:
Noting that more exhaustive documentation for these parameters is captured here: https://github.com/adoptium/aqa-tests/blob/master/doc/pages/JenkinFeatures.md

Reply:
Bear in mind I haven't changed the statement on `CUSTOM_TARGET` in this PR - just moving the quotes in the hope it will expose on the UI what's already there, which does include the - very useful - space-separated example, but it would definitely be great to have some sort of reference to the directory usage too somehow for the casual user! (Either via a link to another doc, or including another example in the list that's already in the parameter description)

> Noting that more exhaustive documentation for these parameters is captured here: https://github.com/adoptium/aqa-tests/blob/master/doc/pages/JenkinFeatures.md

I'm having a look at that page but CUSTOM_TARGET is only referenced in a table that only says `Required when TARGET=jdk_custom` and I can't see a reference to "directory" or "folder" in that doc. If there is a good list of those things it would be great to reference it in the description.

Reply:
Maybe this should be a separate issue/PR now :-)

Reply:
Yes, let's get the EXTRA_OPTIONS and JVM_OPTIONS changes in